azure-pipelines.yml

```diff
@@ -7,10 +7,56 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: 3.6
+    architecture: 'x64'
+
+- task: PythonScript@0
+  displayName: 'Template Runner'
+  inputs:
+    scriptSource: 'inline'
+    script: |
+      """Search all subdirectories for `Runner.py`."""
+      from glob import iglob
+      from os import path
+      import os
+      # Python >= 3.6
+
+      os.environ["PS_BATCH_ACCOUNT_KEY"] = "$(PS_BATCH_ACCOUNT_KEY)" 
```

Comment:
why do you need that for all?

Reply:
I'll remove them

Reply:
so this should be changed to keyvault. but basically i was loading all the configs into the env and reading them that way.